appengine/logs/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
License header? (I think not strictly necessary for config files, but since comment syntax is supported, we have tended to throw them in)